homeassistant/components/kostal_plenticore/translations/fi.json

```diff
@@ -0,0 +1,20 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Laite on jo konfiguroitu"
+        },
+        "error": {
+            "cannot_connect": "Yhdist\u00e4minen ep\u00e4onnistui",
+            "invalid_auth": "Virheellinen kirjautuminen",
+            "unknown": "Odottamaton virhe"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "host": "Is\u00e4nt\u00e4",
+                    "password": "Salasana"
+                }
+            }
+        }
+    }
+}
```

Comment:
Please remove translation changes from this PR.
Translations are managed on lokalize, and then downloaded automatically.

Reply:
Okay, done.